lib/elixir/lib/map.ex

```diff
@@ -599,28 +599,28 @@ defmodule Map do
 
   If `key` is present in `map` with value `value`, `fun` is invoked with
   argument `value` and its result is used as the new value of `key`. If `key` is
-  not present in `map`, `initial` is inserted as the value of `key`. The initial
+  not present in `map`, `default` is inserted as the value of `key`. The default
   value will not be passed through the update function.
```

Comment:
In my mind, `default` would be passed to the function, because that's usually what we expect from `default`? This is probably why `default` wasn't used in the first place?

Reply:
> In my mind, `default` would be passed to the function, because that's usually what we expect from `default`? This is probably why `default` wasn't used in the first place?

I interpret it the exact opposite way. I think of initial as a value that that will change to change,  such as in the initial value of the accumulator in Enum.reduce/2. Default I associate it with the default arg in Map.get, or even  Enum.find/3 uses default, and a function

Reply:
I see. Good points! Note we may need to do similar changes in the Keyword module. :)

Reply:
Sure, I can take care of those changes as well.

Reply:
> I see. Good points! Note we may need to do similar changes in the Keyword module. :)
done
https://github.com/elixir-lang/elixir/pull/10228